tfunc/parse_test.go

```diff
@@ -0,0 +1,35 @@
+package tfunc
+
+import (
+	"bytes"
+	"fmt"
+	"testing"
+
+	"github.com/hashicorp/hcat"
+)
+
+func TestXXXExecute(t *testing.T) {
```

Comment:
This test seems to be WIP? `XXX` and no cases are enumerated

Reply:
Seems the same with `TestTimeExecute` in tfunc/time_test.go

Reply:
I thought I had them all swapped out... weird. Those XXX ones are/were the simple "template" I was copying around to start the tests. I'll get those fixed. Thanks.

Reply:
These are now what they should have been.

Reply:
Thanks.